ts/Series/HeikinAshi/HeikinAshiPoint.ts

```diff
@@ -35,6 +40,9 @@ const {
 
 class HeikinAshiPoint extends CandlestickPoint {
     // clone inheritence
+    public resolveColor(): void {
+        HLCPoint.prototype.resolveColor.call(this);
+    }
```

Comment:
Instead of wrapping and calling the nested function with the same scope, you can just borrow it:

```ts
class HeikinAshiPoint extends CandlestickPoint {
    // clone inheritence
    public resolveColor = HLCPoint.prototype.resolveColor;
}
```